### PR TITLE
#1123 Create owners command.

### PR DESCRIPTION
**Pull request by cmaish**
Implement owners command.

owners command
owners help

### CODE REVIEW

File: JabbR/Nancy/HomeModule.cs [hunk 1]
```diff
@@ -223,6 +223,8 @@ private static string BuildClientResources()
                 "Chat_UserNudgedUser",
                 "Chat_UserLeftRoom",
                 "Chat_YouKickedFromRoom",
+                "Chat_RoomOwnersEmpty",
+                "Chat_RoomOwnersResults",
```
Comment (davidfowl) [inline JabbR/Nancy/HomeModule.cs]: It's time to make this dynamic

**Comment by davidfowl** [comment]
Can we also add who the creator is in the same call?

**Comment by davidfowl** [comment]
ping @cmaish

**Comment by hnrkndrssn** [comment]
Is this ready to be merged?

I can see the creator is now being included, and the resources are dynamic...

**Comment by cmaish** [comment]
Yes, should be good to merge.

**Pull request merged**